tableauserverclient/models/permissions_item.py

```diff
@@ -31,6 +31,7 @@ class Capability:
         ViewUnderlyingData = 'ViewUnderlyingData'
         WebAuthoring = 'WebAuthoring'
         Write = 'Write'
+        Execute = 'Execute'
```

Comment:
Could you move this up to line 23? to keep it alphabetized

Reply:
done!